sprintf/README.md

```diff
@@ -0,0 +1,70 @@
+<a href="https://arcjet.com" target="_arcjet-home">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://arcjet.com/arcjet-logo-dark-planet-arrival.svg">
+    <img src="https://arcjet.com/arcjet-logo-light-planet-arrival.svg" alt="Arcjet Logo" height="144" width="auto">
+  </picture>
+</a>
+
+# `@arcjet/sprintf`
+
+<p>
+  <a href="https://www.npmjs.com/package/@arcjet/sprintf">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/npm/v/%40arcjet%2Fsprintf?style=flat-square&label=%E2%9C%A6Aj&labelColor=000000&color=5C5866">
+      <img alt="npm badge" src="https://img.shields.io/npm/v/%40arcjet%2Fsprintf?style=flat-square&label=%E2%9C%A6Aj&labelColor=ECE6F0&color=ECE6F0">
+    </picture>
+  </a>
+</p>
+
+[Arcjet][arcjet] platform-independent replacement for [util.format][node-util].
```

Comment:
It would be worth noting the rationale for building this - why have we created our own `sprintf` package?

Reply:
Kind of, but it doesn't explain the general rationale e.g. `platform-independent so we can support multiple runtimes in varying environments, such as Edge Runtime, Node.js, Bun, etc.`

Reply:
Okay, but that's the definition of platform-independent 🙃 
```suggestion
[Arcjet][arcjet] platform-independent replacement for [util.format][node-util].

This package is platform-independent in order to support multiple runtimes in varying environments, such as Edge Runtime, Node.js, Bun, Deno, and Cloudflare Workers.
```